bash_script.sh

```diff
@@ -6,38 +6,74 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir task
```

Comment:
Можно в одну команду и с флагом `-p`
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```